src/main/webapp/WEB-INF/views/drivers/all.jsp

```diff
@@ -0,0 +1,43 @@
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>Get all drivers</title>
+</head>
+<body>
+<h1>List of drivers:</h1>
+<style>
+    table, td {
+        border: 1px solid black;
+        border-collapse: collapse;
+        text-align: center;
+    }
+    td {
+        padding: 5px;
+    }
+    tr:nth-child(even) {
+        background-color: #DDDDDD;
+    }
+</style>
```

Comment:
you can create a file  "styles.css" and use here , for example <style><%@include file="/WEB-INF/styles.css"%></style>